web/source/osk/visualKeyboard.ts

```diff
@@ -46,6 +46,12 @@ namespace com.keyman.osk {
   //#region OSK key objects and construction
   export class OSKKeySpec implements keyboards.LayoutKey {
     id: string;
+
+    // Only set (within @keymanapp/keyboard-processor) for keys actually specified in a loaded layout
+    baseKeyIdentifier?: string;
+    coreID?: string;
+    elementID?: string;
```

Comment:
Can you document what these look like with examples here? That will really help for future maintenance.

Also I am not that comfortable with using `Identifier` vs `ID` here. It feels inconsistent.